juniper_official/routing/collect-isis-counters.rule

```diff
@@ -0,0 +1,91 @@
+/*
+ * Collects ISIS lsdb data for seq-num-skips and spf-runs till level-number
+ */
+ healthbot {
+    topic routing.isis {
+        rule collect-isis-counters {
+            keys [ instance-name protocol-name level-num ];
+            synopsis "collects ISIS seq-num-skips,spf-runs";
+            description "Collects LSDB path periodically till level-number node as key";
+            sensor isis-oc {
+            	synopsis "ISIS open-config sensor definition";
+                description "Open-config sensor to collect telemetry data from network device";
+                open-config {
+                    sensor-name /network-instances/network-instance/protocols/protocol/isis/levels/level/link-state-database/;
+                    frequency 60s;
+                }
+            }			
+            field instance-name {
+                sensor isis-oc {
+                    where "/network-instances/network-instance/@name =~ /{{inst-name}}/";
+                    path "/network-instances/network-instance/@name";
+                }			
+                type string;
+                description "instance-name from the lsdb path";
+            }
+            field protocol-name {
+                sensor isis-oc {
+                    where "/network-instances/network-instance/protocols/protocol/@identifier=~/{{proto-name}}/";
+                    path "/network-instances/network-instance/protocols/protocol/@identifier";
+                }			
+                type string;
```

Comment:
What values can come here. Can you give the values examples for us to see if this field is really required?

Reply:
I have removed the protocol-name as key from all the rule files since our path already includes ISIS.